providers/implementations/digests/sha2_prov.c

```diff
@@ -72,6 +72,11 @@ IMPLEMENT_digest_functions(sha256, SHA256_CTX,
                            SHA256_CBLOCK, SHA256_DIGEST_LENGTH, SHA2_FLAGS,
                            SHA256_Init, SHA256_Update, SHA256_Final)
 
+/* ossl_sha256_192_functions */
+IMPLEMENT_digest_functions(sha256_192, SHA256_CTX,
+                           SHA256_CBLOCK, SHA256_192_DIGEST_LENGTH, SHA2_FLAGS,
+                           SHA256_Init, SHA256_Update, SHA256_Final)
+
```

Comment:
Should it be inside `# ifndef FIPS_MODULE` ?

We do not make the implementation available from the fips provider so we should not build it there.

Reply:
Done